src/components/SocialsIcon.js

```diff
@@ -1,12 +1,16 @@
 import styles from '../css/SocialsIcon.module.css'
-import facebook from '../assets/facebook.svg'
-import instagram from '../assets/instagram.svg'
-import github from '../assets/github.svg'
+import facebook from '../assets/socialMediaIcons/facebook.svg'
+import instagram from '../assets/socialMediaIcons/instagram.svg'
+import github from '../assets/socialMediaIcons/github.svg'
+import discord from '../assets/socialMediaIcons/discord.svg'
+import linkedin from '../assets/socialMediaIcons/linkedin.svg'
 
 const socials = [
+  { link: 'https://ca.linkedin.com/company/sfu-surge', img: linkedin },
   { link: 'https://www.facebook.com/sfusurge', img: facebook },
   { link: 'https://www.instagram.com/sfusurge/', img: instagram },
-  { link: 'https://github.com/sfusurge', img: github }
+  { link: 'https://discord.com/invite/hxnB5JPxZF', img: discord },
```

Comment:
Question: This link doesn't ever expire, right?

Reply:
Good question, I actually do not have an answer for this though haha, I copied and pasted the invitation link that Nathan gave me 😅 I'll double check with Naomi or Nathan about this later today!